### ISSUE TITLE
Allow selection of logged in user as a workflow performer

### ISSUE THREAD

**Issue by dgrumieaux** [tsgrp/ActiveWizard#250]
Many client forms involve the form creator as a workflow performer. In order to involve that user in the workflow process, currently we have to configure a question in which the form creator enters his/her own name or email. It would be nice to not require the user to do so.

**Comment by gsteimer** [comment]
We do this in Documentum fairly easily.  As an alternative to this enhancement, simply setup a query like this:

``` sql
select user_login_name, user_name from dm_user where user_name = USER
```

Then, setup the question to default to the query answer.  Since the query always returns one answer, the logged in user will be defaulted.

On Alfresco, we'd just need to create some sort of bean to do a similar thing - get the logged in user and return his/her info.

**Comment by dhartman25** [comment]
Closed with commit #12250
